javacomparator/OoxmlFile.java

```diff
@@ -0,0 +1,65 @@
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.io.File;
+import java.io.FileReader;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+
+/**
+ * OoxmlFile Class implement various file loading methods to load JSON files into the Objects
+ */
+public class OoxmlFile {
+    String fileDataPath;
+    boolean roundtripped;
+
+    Map<String, JSONObject> jsonDataFiles;
+
+    /**
+     * Constuctor takes in the FolderPath for the file and creates a list of files to be compared.
+     *
+     * @param folderPath   Folder path that has the OOXML content of the file in JSON format
+     * @param roundtripped Whether the file is roundTripped file or not.
+     */
+    public OoxmlFile(String folderPath, boolean roundtripped) {
+        this.fileDataPath = folderPath;
```

Comment:
You can have same names for constructor parameters and class variable names. Just use ```this``` operator.